students/piechowski_michal/lesson_12_debugging/validated_user_base.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+import argparse
+import csv
+import sys
+
+from students.piechowski_michal.lesson_08_regular_expressions.email_validator\
+    import is_valid as is_valid_email
+from students.piechowski_michal.lesson_08_regular_expressions.phone_number_validator\
+    import is_valid as is_valid_phone_number
+from students.piechowski_michal.lesson_08_regular_expressions.postal_code_validator\
+    import is_valid as is_valid_postal_code
+from students.piechowski_michal.lesson_08_regular_expressions.strong_password_detection\
+    import is_strong as is_strong_password
+
+
+verbosity = ""
+
+
+def my_print(tag, text):
```

Comment:
w tych zadaniach raczej chodziło, o wykorzystanie modułu `logging` i sterowanie nim za pomocą argumentów np:
przy uruchomieniu user podaje: `nazwa_skryptu.py -L debug` i zbierane są logi zgodne z tym poziomem. 
Możliwe jest logowanie albo do konsoli (`StreamHandler`) lub do pliku (`FileHandler`), w bardziej rozbudowanej wersji można zrobić, że jak user nie poda ścieżki do pliku z logami, to logujemy do konsoli, a jak poda to do podanego pliku.
W prostszej wersji wybierz jeden wariant.

Reply:
Ok